src/Routes/TrustScroll/TrustScroll.js

```diff
@@ -0,0 +1,102 @@
+const DataProtection = lazy(() => import(/* webpackChunkName: "DataProtection" */ '../../Components/Sections/DataProtection'));
+const FAQ = lazy(() => import(/* webpackChunkName: "FAQ" */ '../../Components/Sections/FAQ'));
+
+const TrustScroll = () => {
+    const intl = useIntl();
+
+    const jumpLinks = [{
+        key: 0, section: 'overview', title: intl.formatMessage(messages.overview),
+        component: <Suspense fallback={<Loading />}> <Overview /> </Suspense>
+    }, {
+        key: 1, section: 'data-collection', title: intl.formatMessage(messages.dataCollection),
+        component: <Suspense fallback={<Loading />}> <DataCollection /> </Suspense>
+    }, {
+        key: 2, section: 'data-protection', title: intl.formatMessage(messages.dataProtection),
+        component: <Suspense fallback={<Loading />}> <DataProtection /> </Suspense>
+    }, {
+        key: 3, section: 'faq', title: intl.formatMessage(messages.faq),
+        component: <Suspense fallback={<Loading />}> <FAQ /> </Suspense>
+    }
+    ];
+
+    return <div className="ins-c-marketing-page">
+        <PageHeader className='ins-c-page-header' isWidthLimited>
+            <Flex justifyContent={{ default: 'justifyContentCenter' }}>
+                <TextContent
+                    className='ins-c-width-limiter pf-u-text-align-center'
+                    style={ { '--ins-c-width-limiter--MaxWidth': '80ch' } }>
+                    <Title headingLevel='h1' size={TitleSizes['3xl']}>
+                        {intl.formatMessage(messages.redHatInsightsSecurity)}
+                    </Title>
+                    <Text>
+                        {intl.formatMessage(messages.redHatInsightsSecurityBody)}
+                    </Text>
+                    <Button className='ins-c-dashboard-link' component='a' variant='primary' href='https://cloud.redhat.com/insights/' >
+                        {intl.formatMessage(messages.goto)}
+                    </Button>
+                </TextContent>
+            </Flex>
+        </PageHeader>
+        <PageSection className="pf-u-pt-sm pf-u-pb-0" sticky="top">
+            <JumpLinks isCentered scrollableSelector="#root">
+                {jumpLinks.map(item =>
+                    <JumpLinksItem key={item.key} href={`#section-${item.section}`}>
+                        {item.title}
+                    </JumpLinksItem>
+                )}
+            </JumpLinks>
+        </PageSection>
+        <PageGroup hasOverflowScroll id="trust-scroll">
+            {jumpLinks.map(item =>
+                <PageSection key={item.key}>
+                    <div className="pf-c-page__anchor" id={`section-${item.section}`} tabIndex={-1}></div>
+                    <Flex justifyContent={{ default: 'justifyContentCenter' }}>
+                        <FlexItem
+                            className='ins-c-width-limiter'
+                            style={ { '--ins-c-width-limiter--MaxWidth': '80ch' } }>
```

Comment:
prefer the use of classes to inline styles 😬

Reply:
Agree! In this case, I'm only updating a css variable so that the `width-limiter` component is reusable and doesn't have tons of modifier classes (like `ins-m-max-width-80ch`, `ins-m-max-width-90ch`, `ins-m-max-width-12ch`, etc), so you can do this:
```
style={ {
  '--ins-c-width-limiter--MaxWidth': '10ch' ,
  '--ins-c-width-limiter--MaxWidth-on-sm': '120px' ,
  '--ins-c-width-limiter--MaxWidth-on-md': '12rem' ,
  '--ins-c-width-limiter--MaxWidth-on-lg': '50%' } }>
```

Essentially, it can have any width, at any of our predefined PF breakpoints. However, if this approach is a non-starter, I can remove 😄

Reply:
Ah okie dokie, gooood point, ok down with it 👍 🦖